OmniBLE/PumpManagerUI/ViewModels/InsertCannulaViewModel.swift

```diff
@@ -124,6 +124,14 @@ class InsertCannulaViewModel: ObservableObject, Identifiable {
     }
 
     @Published var state: InsertCannulaViewModelState = .ready
+    public var stateNeedsCriticalUserAcceptance : Bool {
```

Comment:
nit: "Critical" is an adverb we usually use to describe the severity of a situation or alert. In this case (based on the PR description) I'm guessing you're wanting to convey the need for "intentional" or "deliberate" action on the part of the user, so maybe `stateNeedsCriticalUserAcceptance` -> `stateNeedsDeliberateUserAcceptance`